### ISSUE TITLE
Implement hard sigmoid and hard swish

### ISSUE THREAD

**Issue by joaopauloschuler** [joaopauloschuler/neural-api#67]
Use this code as inspiration:
def HardSigmoid(x):
    return tensorflow.keras.layers.ReLU( 6.0 )( x + 3.0 ) * ( 1.0 / 6.0 )

def HardSwish(x):
    return tensorflow.keras.layers.Multiply()([tensorflow.keras.layers.Activation(HardSigmoid)(x), x])
